toolkit/scripts/toolchain/container/Dockerfile

```diff
@@ -66,6 +66,7 @@ COPY [ "./toolchain-md5sums", \
 WORKDIR $LFS/sources
 RUN wget -nv --no-clobber --timeout=30 --no-check-certificate --continue --input-file=$LFS/tools/toolchain-local-wget-list --directory-prefix=$LFS/sources; exit 0
 RUN wget -nv --no-clobber --timeout=30 --continue --input-file=$LFS/tools/toolchain-remote-wget-list --directory-prefix=$LFS/sources; exit 0
+RUN wget -nv --no-clobber --timeout=30 --continue https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/mariner/5.10.13.1.tar.gz -O kernel-5.10.13.1.tar.gz --directory-prefix=$LFS/sources; exit 0
```

Comment:
Could you also add a comment explaining why we download this one file here and not through the `toolchain-remote-wget-list`? I understand it's needed but we may later forget and a comment might save us some time in the future.

Reply:
Added comment